Ark.Tools.FtpClient.ArxOne/FtpClientPoolArxOneWithSocksFactory.cs

```diff
@@ -28,7 +22,7 @@ public IFtpClientPool Create(int maxPoolSize, FtpConfig ftpConfig)
             EnsureArg.IsNotNull(ftpConfig.Uri);
             EnsureArg.IsNotNull(ftpConfig.Credentials);
 
-            return new FtpClientPoolArxOneWithSocks(_config, maxPoolSize, ftpConfig, _configurer);
+            return new FtpClientPoolArxOneWithSocks(_config, maxPoolSize, ftpConfig);
```

Comment:
I don't see expect to see the FtpClientPool_WithSocks anymore. All 3 factories should return an instance of the same Pool class. the 3 factories use 3 different means to obtain the IConfig, and call different ctors on tbe FtpClientPoolTheNewOne (find a nice name).

Reply:
I replaced FtpClientPoolArxOneWithSocks and FtpClientPoolArxOneWithSocksAndConfigurer with overloaded ctors in the FtpClientPoolArxOne class. That seemed like the simplest naming convention.